### DISCUSSION TITLE
Rebranding Plan for RingDAO Ecosystem

### DISCUSSION THREAD

**Discussion by hackfisher** [ringecosystem/discussion#1] [Announcements]
# Rebranding Plan for RingDAO Ecosystem

**Objective:**  
Rebrand and unify the branding and media assets across the RingDAO ecosystem, including Darwinia, Msgport, and XAPI, while maintaining RING as the governance token.

---

## 1. Website and Domain Strategy

- [x] **Primary Domain:**
  - Use **ringdao.com** as the main website for the entire ecosystem.

- [ ] **Subdomains for Specific Projects:**
  - [x] **Msgport:** Transition to **msgport.ringdao.com**. Redirect the existing domains:
    - **https://msgport.xyz/** to **msgport.ringdao.com**
    - **https://docs.msgport.xyz/** to **msgport.ringdao.com/docs**
  - [ ] **XAPI:** Transition to **xapi.ringdao.com**.
  
- [x] **Darwinia Chain:**
  - Keep **darwinia.network** website unchanged to preserve its existing branding and utility.

---

## 2. GitHub Reorganization

- [x] **RingDAO Ecosystem:**
  - [x] Rename the **https://github.com/xapi-box** repository to **https://github.com/ringecosystem**. This will serve as the central repository for RingDAO ecosystem projects, including Msgport and XAPI.
  - [x] Migrate repositories from **https://github.com/msgport** to the newly named **ringecosystem** organization and deprecate **https://github.com/msgport**.
  - [x] Migrate part of the repositories not related to Darwinia chain from **https://github.com/darwinia-network** to the newly named **ringecosystem** organization.

- [x] **Darwinia Network:**
  - The **https://github.com/darwinia-network** repository will remain unchanged to maintain continuity for Darwinia chain.

---

## 3. Social Media Rebranding

- [x] **X Account (formerly Twitter):**
  - Rebrand and rename the existing Msgport account **(https://x.com/msgport_xyz)** to **(https://x.com/ringecosystem)** to represent the broader ecosystem.

- [x] **Telegram:**
  - Rebrand and rename the Msgport Telegram to **https://t.me/ringecosystem** to reflect the unified branding of the ecosystem.
  
- [x] **Discord**
  - Rename the Darwinia Discord server to RingDAO, and replace server's logo with RingDAO's logo 

---

## 4. Communication Strategy

- **Announcement:** 
  - Develop a comprehensive communication plan to inform the community about the rebranding, ensuring clarity on the continuity of RING’s utility and governance role across all projects.
  - Use all social media channels, including the new RingDAO X account and Telegram, to disseminate information.

- **Documentation and FAQs:**
  - Update documentation across all projects to reflect the rebranding and provide FAQs to assist users with the transition.

---

**Timeline:**  
Implement these changes over the next week, ensuring minimal disruption to the community and operations.

**Review:**  
Conduct a post-implementation review to ensure all assets are correctly branded and functioning as intended.

**Comment by Divblaze** [comment]
MSGport Telegram community is less active so it nice the transition is coming now before more users come in
